drivers/SmartThings/philips-hue/src/disco.lua

```diff
@@ -20,6 +20,8 @@ local supported_resource_types = {
   light = true
 }
 
+local try_create_count = {}
```

Comment:
does this get used for anything except logging?

Reply:
This is only used for logging, but it's *really* easy for a Hue setup to be in the rate limiting regime in the hub-core firmware for `try_create_device` (1-per-second sustained, up to 10 per second in a burst). It's an odd heuristic so I was wanting to leave this in to capture some data about how this impacts people.

The impetus is that we get a lot of reports of "not all devices get added when I scan nearby" if you have a metric ton of devices attached to your bridge, and I want to get metrics on how to make that process smoother.